homeassistant/components/sensor/onewire.py

```diff
@@ -42,20 +51,40 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAMES): {cv.string: cv.string},
     vol.Optional(CONF_MOUNT_DIR, default=DEFAULT_MOUNT_DIR): cv.string,
+    vol.Optional(CONF_SERVER_HOST, default=DEFAULT_SERVER_HOST): cv.string,
+    vol.Optional(CONF_SERVER_PORT, default=DEFAULT_SERVER_PORT): cv.port,
+    vol.Optional(CONF_SERVER_TIMEOUT,
+                 default=DEFAULT_SERVER_TIMEOUT): cv.positive_int,
 })
 
 
 # pylint: disable=unused-argument
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the one wire Sensors."""
     base_dir = config.get(CONF_MOUNT_DIR)
+    server_host = config.get(CONF_SERVER_HOST)
+    server_port = config.get(CONF_SERVER_PORT)
+    server_timeout = config.get(CONF_SERVER_TIMEOUT)
     devs = []
```

Comment:
Not your code but could you please give that variable a real name like `devices`. Thanks.